src/status_im/ui/screens/wallet/history/views.cljs

```diff
@@ -126,30 +126,30 @@
 
 (defn- token->icon [s]
   (case s
-    "GNO" :dollar_green
-    :ok_blue))
+    "GNO" [:icons/ok {:color :green}] ;TODO :dollar
+    [:icons/ok {:color :blue}]))
 
 (defn item-tokens [{:keys [symbol label]}]
   [list/item
-   [list/item-icon (token->icon symbol)]
-   [list/item-content label symbol]
+   ((comp vec flatten vector) list/item-icon (token->icon symbol))
+   [list/item-content label symbol]])
    ;; TODO checkbox
-   ])
+
 
 (defn- type->icon [k]
   (case k
-    "incoming"  :dollar_green
-    "outgoing"  :ok_blue
-    "pending"   :dollar_green
-    "postponed" :ok_blue
-    :ok_blue))
+    "incoming"  [:icon/ok {:color :blue}] ;TODO :dollar_green
```

Comment:
Does this mean there's a regression with dollar_green?

Reply:
yes, we need dollar svg from designers, i'll ask maybe include it in this PR

Reply:
andmironov [8:43 PM] 
i believe the icon got here from older versions somehow (or from website?) we do not need it anyway